src/actions/clientActions.js

```diff
@@ -0,0 +1,22 @@
+import { CLIENT_CHANGE } from '../constants/actionTypes'
+
+function isSmallScreen () {
+  return global.window.innerWidth <= 768
+}
+
+function clientChange (smallScreen) {
+  return {
+    type: CLIENT_CHANGE,
```

Comment:
Well, the relevant property in global state is `client`; the idea is to have a central source of truth about client features which are relevant for app. In some components this is still handled under `isMobile` boolean, but that is misleading.

I'm not saying this is the best name, but it makes sense to me in this context. Feel free to propose different names for both reducer and associated actions.